### PR TITLE
Refactor/clean UI and synchronize with figma

### PR DESCRIPTION
**Pull request by misterM125**
**Overview**

This PR aims to clean the overall app ui by changing colors to adhere to app themes and modify certain screens to adress some UI issues

**implementation**

- Most screens have been changed to use colors from the `ColorPalette` class
- The park overview "create event button has been moved to not overlap with the event list
- image has been removed from the event overview and park details has been centered to cover empty space
- event title will now be displayed underneath the top app bar in the event overview screen
- Some tests have been refactored in consequence

Screenshot from 2024-11-13 20-52-54
Screenshot from 2024-11-13 20-52-10

### CODE REVIEW

**Review by kzepfl — COMMENTED** [review]
The PR is great Most of the UI is identical to the Figma, i have just some suggestion that i commented

**Review by kzepfl — APPROVED** [review]
The PR is great most of the UI is identical to the Figma, i have just some suggestion, see if you want to change it. 
Some element size is not like the FIgma but it can be refractor in the future if we want it to be just like the Figma

File: app/src/main/java/com/android/streetworkapp/ui/profile/AddFriend.kt [hunk 1]
```diff
@@ -81,6 +82,7 @@ fun AddFriendScreen(
                   Toast.makeText(context, "Friend request sent.", Toast.LENGTH_SHORT).show()
                 }
               },
+              colors = ColorPalette.BUTTON_COLOR,
               modifier = Modifier.size(220.dp, 50.dp).testTag("RequestButton")) {
```
[the remark this answers is not in the document]
Comment (misterM125) [reply app/src/main/java/com/android/streetworkapp/ui/profile/AddFriend.kt]: Fixed for both, thank you for noticing it!

File: app/src/main/java/com/android/streetworkapp/ui/profile/NfcButton.kt [hunk 2]
```diff
@@ -8,10 +8,14 @@ import androidx.compose.ui.Modifier
...
+import com.android.streetworkapp.ui.theme.ColorPalette
 
 @Composable
 fun NfcButton() {
-  Button(onClick = {}, modifier = Modifier.size(220.dp, 50.dp).testTag("NFCButton")) {
-    Text(text = "Activate NFC", fontSize = 17.sp)
-  }
+  Button(
+      onClick = {},
+      modifier = Modifier.size(220.dp, 50.dp).testTag("NFCButton"),
```
[the remark this answers is not in the document]
Comment (misterM125) [reply app/src/main/java/com/android/streetworkapp/ui/profile/NfcButton.kt]: same here

**Pull request merged**